ui-lib/src/main/java/co/electriccoin/zcash/ui/MainActivity.kt

```diff
@@ -42,7 +43,7 @@ import kotlin.time.Duration
 import kotlin.time.Duration.Companion.milliseconds
 import kotlin.time.Duration.Companion.seconds
 
-class MainActivity : ComponentActivity() {
+class MainActivity : FragmentActivity() {
```

Comment:
Will this change affect composability when assembling the project for desktop application?

Reply:
To support BioMetrics we need to change this